cmsplugin_carousel_ai/models.py

```diff
@@ -0,0 +1,89 @@
+# -*- coding: utf-8 -*-
+from django.db import models
+from django.utils.translation import ugettext_lazy as _
+
+from cms.models import CMSPlugin, Page, force_text
+from easy_thumbnails.files import get_thumbnailer
```

Comment:
Looks like an unused import. Have you run the usual code style tools on the source code?